parsl/providers/cobalt/cobalt.py

```diff
@@ -196,7 +197,10 @@ def submit(self, command, tasks_per_node, job_name="parsl.cobalt"):
             self.resources[job_id] = {'job_id': job_id, 'status': JobStatus(JobState.PENDING)}
         else:
             logger.error("Submission of command to scale_out failed: {0}".format(stderr))
-            raise (ScaleOutFailed(self.__class__, "Request to submit job to local scheduler failed"))
+            raise SubmitException(job_name,
```

Comment:
You should still report the class name so that you can identify which provider is failing.